scripts/prepare-host.sh

```diff
@@ -5,11 +5,6 @@ export $(echo $(cat ${PROJECT_PATH}/.env | sed 's/#.*//g'| xargs -0) | envsubst)
 
 set -ex
 
-# superset
```

Comment:
are you planning to move the core data from sqlite to postgresql?

Reply:
According to this issue, superset is dropping support for SQLite https://github.com/apache/superset/issues/9748

Reply:
Got it. Thanks for the explanation